src/app/public/modules/demo/demo.component.spec.ts

```diff
@@ -9,6 +13,10 @@ import {
   expect
 } from '@skyux-sdk/testing';
 
+import {
+  SkyRestrictedViewAuthService
+} from '@blackbaud/skyux-lib-restricted-view/modules/restricted-view/restricted-view-auth.service';
```

Comment:
Is this type not exported at the root? If not, I definitely need to address this deep import for SKY UX 4.

Reply:
It isn't. I'm importing it because I need to mock it out, but it's really an internal implementation of the `sky-restricted-view` component. Maybe a safer option would be to go one level deeper and mock out `SkyAuthTokenProvider` instead, since `SkyRestrictedViewAuthService` calls into it. I'll look into doing that.